### PR TITLE
Update fingerptin structure

### PR DESCRIPTION
**Pull request by mohsinenar**

### CODE REVIEW

File: src/ostorlab/agent/message/proto/v3/fingerprint/file/android/library/library.proto [hunk 1]
```diff
@@ -4,8 +4,10 @@ package ostorlab.agent.message.proto.v3.fingerprint.file.android.library;
 
 message Message {
   optional string path = 1;
-  optional string library_name = 2;
-  optional string library_version = 3;
-  optional string library_type = 4;
-  optional string detail = 5;
+  required string package_name = 3;
```
Comment (amine3) [inline src/ostorlab/agent/message/proto/v3/fingerprint/file/android/library/library.proto:7]: where is number 2?

**Review by amine3 — CHANGES_REQUESTED** [review]

File: src/ostorlab/agent/message/proto/v3/fingerprint/file/android/library/library.proto (hunk 1 above)
Comment (mohsinenar) [reply src/ostorlab/agent/message/proto/v3/fingerprint/file/android/library/library.proto]: Fixed.

File: src/ostorlab/agent/message/proto/v3/fingerprint/file/ios/feature/feature_pb2.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: src/ostorlab/agent/message/proto/v3/fingerprint/file/ios/feature/feature.proto
```
Comment (amine3) [inline src/ostorlab/agent/message/proto/v3/fingerprint/file/ios/feature/feature_pb2.py:3]: Protos should be compiled from src and the path should ostorlab/agent/message/proto/v3/fingerprint/file/ios/feature/feature.proto

**Review by amine3 — CHANGES_REQUESTED** [review]

File: src/ostorlab/agent/message/proto/v3/fingerprint/file/android/feature/feature.proto [hunk 3]
```diff
@@ -0,0 +1,11 @@
+syntax = "proto2";
+
+package ostorlab.agent.message.proto.v3.fingerprint.file.android.feature;
+
+message Message {
+  optional string path = 1;
+  required string package_name = 2;
+  optional string feature_name = 3;
+  optional string feature_description = 4;
+  repeated string detail = 5;
```
Comment (oussamaessaji) [inline src/ostorlab/agent/message/proto/v3/fingerprint/file/android/feature/feature.proto:10]: it shouldn't be feature_detail as the prefix here is feature??

File: src/ostorlab/agent/message/proto/v3/fingerprint/file/feature/feature.proto [hunk 4]
```diff
@@ -3,7 +3,8 @@ syntax = "proto2";
...
 message Message {
-  optional string feature_name = 1;
-  optional string feature_description = 2;
-  repeated string detail = 3;
-}
\ No newline at end of file
+  optional string path = 1;
+  optional string feature_name = 2;
+  optional string feature_description = 3;
+  repeated string detail = 4;
```
Comment (oussamaessaji) [inline src/ostorlab/agent/message/proto/v3/fingerprint/file/feature/feature.proto:9]: same here!

File: src/ostorlab/agent/message/proto/v3/fingerprint/file/ios/library/library.proto [hunk 5]
```diff
@@ -4,8 +4,9 @@ package ostorlab.agent.message.proto.v3.fingerprint.file.ios.library;
...
   optional string path = 1;
-  optional string library_name = 2;
-  optional string library_version = 3;
-  optional string library_type = 4;
-  optional string detail = 5;
+  required string bundle_id = 2;
+  optional string library_name = 3;
+  optional string library_version = 4;
+  optional string library_type = 5;
+  optional string detail = 6;
```
Comment (oussamaessaji) [inline src/ostorlab/agent/message/proto/v3/fingerprint/file/ios/library/library.proto:11]: ??

File: src/ostorlab/agent/message/proto/v3/fingerprint/file/android/feature/feature.proto (hunk 3 above)
Comment (3asm) [reply src/ostorlab/agent/message/proto/v3/fingerprint/file/android/feature/feature.proto:10]: +1.

**Review by 3asm — CHANGES_REQUESTED** [review]

**Review by amine3 — APPROVED** [review]

**Review by oussamaessaji — APPROVED** [review]

**Review by 3asm — APPROVED** [review]

**Pull request merged**